dali/python/nvidia/dali/backend.py

```diff
@@ -38,5 +38,13 @@
                           "reaches the end of life on January 1st, 2020. To stay up to date with "
                           "DALI, please upgrade to Python 3.5 or later.", Warning, stacklevel=2)
 
+    # TODO: remove in 0.20
+    def old_at(self, id):
+        warnings.warn("`TensorListGPU.at` is deprecated for `TensorGPUList.tensor`. "
```

Comment:
It is to widely used to deprecate it so soon, if at all.

Reply:
"In a future version" would be enough?

Reply:
Sounds better for me

Reply:
Done